$MirOS: ports/www/firesomething/patches/patch-widget_src_gtk2_nsSound_cpp,v 1.3 2008/05/21 06:31:05 bsiegert Exp $
$OpenBSD: patch-widget_src_gtk2_nsSound_cpp,v 1.1 2005/01/04 21:27:28 wilfried Exp $
--- widget/src/gtk2/nsSound.cpp.orig	2007-05-20 10:56:36.000000000 +0200
+++ widget/src/gtk2/nsSound.cpp	2008-08-24 14:52:52.000000000 +0200
@@ -115,7 +115,7 @@ nsSound::Init()
 
     EsdOpenSoundType EsdOpenSound;
 
-    elib = PR_LoadLibrary("libesd.so.0");
+    elib = PR_LoadLibrary("libesd.so");
     if (!elib) return NS_ERROR_FAILURE;
 
     EsdOpenSound = (EsdOpenSoundType) PR_FindFunctionSymbol(elib, "esd_open_sound");
